sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 12.8.1 (Unreleased)
+## 12.9.0b1 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for service version 2021-08-06.
```

Comment:
We didn't actually add support for the new version in this release since there were no Swagger changes. You can remove this line.